sap-s4hana-soap/2.3/modules/ROOT/pages/sap-s4hana-soap-connector-reference.adoc

```diff
@@ -70,14 +70,46 @@ Basic authentication is the simplest way of enforcing access controls because it
 ** SECONDS
 ** MINUTES
 ** HOURS
-** DAYS |  Time unit to use in the Timeout configuration |  `SECONDS` |
+** DAYS |  Time unit to use in the timeout configuration |  `SECONDS` |
 | Default HTTP headers a| Object | Default HTTP headers to include in the message. |  |
 | Default query parameters a| Object | Default query parameters to include in the request. |  |
 | Proxy configuration a| <<ProxyConfiguration>> |  Proxy configuration for the connector. |  |
 | TLS configuration a| <<Tls>> |  Defines a configuration for TLS, which can be used from both the client and server sides to secure communication for the Mule app. When using the HTTPS protocol, the HTTP communication is secured using TLS or SSL. If HTTPS is configured as the protocol then the user needs to configure at least the keystore in the `tls:context` child element of the `listener-config`. | HTTP |
 | Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
 |===
 
+[[Config_CertificateAuthentication]]
+===== Certificate Authentication
+
+An X.509 certificate is a digital certificate based on the widely accepted International Telecommunications Union (ITU) X.509 standard, which defines the format of public key infrastructure (PKI) certificates. They are used to manage identity and security in internet communications and computer networking. X.509 authentication requires a discovery service configuration, a valid certificate and a secure connection (HTTPS).
+
+====== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Service base URL a| String |  The base URL of the service on SAP S/4HANA to be accessed from the connector. |  | x
+| Discovery Service Enabled a| Boolean |  Enables automatic service discovery |  true |
```

Comment:
Usually we want the parameter descriptions to all end with periods and not start with "the", but this can be something to keep in mind when working on the reference cleanup

Reply:
I think I have time today, so I'll go ahead and seize the opportunity.